packages/editor/src/components/post-slug/test/check.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * External dependencies
+ */
+import { shallow } from 'enzyme';
+
+/**
+ * Internal dependencies
+ */
+import { PostSlugCheck } from '../check';
+
+describe( 'PostSlugCheck', () => {
+	it( 'should render control', () => {
```

Comment:
Should we test for when there is no support and it should not render?

Reply:
I don't think that is necessary, as we would be testing the `PostTypeSupportCheck` more than `PostSlugCheck`.

Reply:
Fair enough.